README.md

```diff
@@ -1 +1,197 @@
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **ES6 Syntax**
+- **DOM Manipulatio**
+- **Webpack**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+1. Go to this URL link: https://github.com/hamzaw9/ToDoList
+2. clone the repo and start working on it.
+
+### Prerequisites
+
+Google Chrome or other browser
+
+```sh
+  https://www.google.com/chrome/?brand=JJTC&gclid=CjwKCAjw9J2iBhBPEiwAErwpeSDcMFWiIQWj2u5GY6owZ7OaOHw7dYYCHW7uTR4kvYosNJYd4wt4VxoCiywQAvD_BwE&gclsrc=aw.ds
+```
+
+Github Account:
+
+```sh
+  https://github.com/
+```
+
+Npm installed:
+
+```sh
+  https://nodejs.org/en/download
+```
+
+Git installed:
+
+```sh
+  https://git-scm.com/downloads/
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/hamzaw9/ToDoList
+```
+
+### Install
+
+Install this project with:
+
+Linters:
+
+Lighthouse run:
+
+```sh
+  npm install -g @lhci/cli@0.7.x
+```
+
+Webhint run
+
+```sh
+ npm install --save-dev hint@7.x
+```
+
+StyleLint run:
+
+```sh
+ npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  cd bookListManager
+  index.html
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx hint .
+```
+
+### Deployment
+
+You can deploy this project using:
+
+Your working browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors">Hamza Baig</a>
+
+👤 **Author**
+
+- GitHub: [@hamzabaig](https://github.com/hamzaw9)
+- Twitter: [@hamzabaig](https://twitter.com/hamzabaig059)
+- LinkedIn: [@hamzabaig](https://www.linkedin.com/in/hamza-baig-852058269/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Frontend Framework**
+- [ ] **Database**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/hamzaw9/ToDoList/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you find this project useful and informative, please consider giving it a star. Your support and feedback are greatly appreciated and will motivate me to continue creating high-quality projects in the future.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank **Microverse**.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ Required ] I think it would be better if you add the `LICENSE` file in the root directory of the repository and update the link of the `LICENSE` on the readme file. It's very important to add a License to the project because it will serve to inform someone who would like to use code from your project how to use it within the law.